### ISSUE TITLE
Add rubocop instructions to README

### ISSUE THREAD

**Issue by conroywhitney** [roguerb/pull-request-workshop#2]
Since we require that all PRs conform to our coding styles, we should provide instructions for how to run `rubocop` in the `README` file.

**Comment by programmin1** [comment]
What's more, Rubocop gives an error with it currently:

```
rogue_math.rb:2:19: W: Lint/UnusedMethodArgument: Unused method argument - b. If it's necessary, use _ or _b as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
  def self.add(a, b)
                  ^

```

I guess that should be a + b in adder ?

**Comment by conroywhitney** [comment]
Hey @programmin1. Thanks for pointing this out!

You're right -- our project doesn't currently even pass our own style guides!

Could you please create a separate issue called something like "Fix linter warnings" so that we can work on fixing this?

Thanks!

**Comment by conroywhitney** [comment]
@programmin1 Haha, I was too slow. I just saw your PR #6 and left some feedback there. Thanks!

**Comment by programmin1** [comment]
Thanks @conroywhitney, please see  #8

**Comment by conroywhitney** [comment]
Hey @programmin1. I actually just recently closed #8 and left some comments there as to why. Thanks!

**Comment by conroywhitney** [comment]
Closing RE: PR #14
